openAI.sublime-settings

```diff
@@ -1,4 +1,10 @@
 {
+    // Custom URL for API (useful when using custom, OpenAI compattible API) in format host:port
+    "url": "api.openai.com",
+    
+    // Whether to use HTTPS for connection
+    "https": true,
```

Comment:
I believe we could avoid this by going a bit further work in the url setup. By parsing the string from `"url"` property and managing the control flow based on whether there `http` or `https` appears in the url provided by a user (the URL should be stored with such prefix ofc, and maybe should be dropped within the code itself (just dunno is it necessary for `HTTP(S)Connection` init). 

Just in case you'll face any struggle with implementing this please feel free to reach out for any help you'd need.